pool/paymentmgr.go

```diff
@@ -896,6 +902,34 @@ func (pm *PaymentMgr) applyTxFees(inputs []chainjson.TransactionInput, outputs m
 	return sansFees, estFee, nil
 }
 
+// fetchTxConfNotifications is a helper function for fetching tx confirmation
+// notifications without blocking.
```

Comment:
this comment is a bit misleading because it says "without blocking", but actually `fetchTxConfNotifications` is a blocking func.

How about...

> fetchTxConfNotifications is a helper function for fetching tx confirmation notifications. It will return when either a notification or error are received from the provided notification source, or when the provided context is cancelled.

Reply:
Noted, meant to say it doesn't block on fetching tx confirmation notifications.